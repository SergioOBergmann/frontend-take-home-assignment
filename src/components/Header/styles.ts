import styled from 'styled-components';

import { MOBILE_BREAKPOINT } from '../../styles/global';

export const Logo = styled.img`
  width: 95px;
  margin-left: 37px;

  @media (max-width: ${MOBILE_BREAKPOINT}px) {
    width: 65px !important;
    margin-left: 16px !important;
  }
`;

export const Bg = styled.div`
  align-items: center;
  width: 100%;
  height: 79px;
  background: #ffffff;

  @media (max-width: ${MOBILE_BREAKPOINT}px) {
    height: 56px !important;
  }
`;
