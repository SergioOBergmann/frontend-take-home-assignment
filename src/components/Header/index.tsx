import * as React from 'react';

import originLogo from './../../icons/logo.svg';
import { Bg, Logo } from './styles';

const Header: React.FC = () => {
  return (
    <>
      <Bg>
        <Logo src={originLogo} alt="Origin" />
      </Bg>
    </>
  );
};

export default Header;
