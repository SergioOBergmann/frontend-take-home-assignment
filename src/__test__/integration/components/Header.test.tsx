import * as React from 'react';

import { render, RenderResult } from '@testing-library/react';

import Header from '../../../components/Header';

let documentBody: RenderResult;

describe('<Header />', () => {
  beforeEach(() => {
    documentBody = render(<Header />);
  });

  it('matches snapshot', () => {
    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
