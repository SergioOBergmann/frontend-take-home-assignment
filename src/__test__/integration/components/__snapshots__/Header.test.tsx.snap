// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-htoDjs idzlLt"
    >
      <img
        alt="Origin"
        class="sc-gzVnrw iCOvDr"
        src="test"
      />
    </div>
  </div>
</body>
`;
